lib/websocket_client.js

```diff
@@ -1,89 +1,119 @@
-let web_socket = require("ws");
+let web_socket = require('ws');
 let atob = require('atob');
 let pako = require('pako');
 
 let triggers = {
-     "connect": [],
-     "tick": []
+	connect: [],
+	tick: [],
 };
 
 let WebSocketClient = function (params) {
-     let self = this;
+	let self = this;
 
-     let ws = null,
-          clientcode = params.clientcode || null,
-          jwttoken = params.jwttoken || null,
-          apikey = params.apikey || null;
-          feedtype = params.feedtype || null;
+	let ws = null,
+		clientcode = params.clientcode || null,
+		jwttoken = params.jwttoken || null,
+		apikey = params.apikey || null;
+	feedtype = params.feedtype || null;
 
-     let url = params.url || "wss://smartapisocket.angelbroking.com/websocket?jwttoken=" + jwttoken + "&&clientcode=" +clientcode+ "&&apikey=" +apikey;
+	let url =
```

Comment:
why changes are made to this file?

Reply:
Changed happend due to visual studio formatter , it won't affect the code flow